docker/DockerImages.md

```diff
@@ -0,0 +1,25 @@
+# Docker Images #
+
+The following docker images have been verified to successfully build the source-build repo.
+
+| OS | Image |
+| --- | --- |
+| CentOS 7.1 | microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416 |
+| Fedora 28 | microsoft/dotnet-buildtools-prereqs:fedora-28-c103199-20180628122443 |
+| Ubuntu 16.04 | microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-c103199-20180628134544 |
+
+## Building source-build with Docker ##
+
+```
+> ### Create a volume on which to build source-build
+> docker volume create source-build-volume 
+
+> ### Clone source-build repo
+> docker run -it --rm --mount source=source-build-test,target=/src -w /src <imageName> /bin/bash -c "git clone https://github.com/dotnet/source-build"
+
+> ### Build source-build to create a tarball
+> docker run -it --rm --mount source=source-build-test,target=/src -w /src/source-build <imageName> /bin/bash -c "./build-source-tarball.sh ../tarball"
+
+> ### Build the tarball
+> docker run -it --rm --mount source=source-build-test,target=/src -w /src/tarball <imageName> /bin/bash -c "./build.sh"
```

Comment:
I feel like it would be more useful to give the reader something easy to understand at first glance, and they can use more Docker features if they want to. How about:

```sh
### Run a bash shell inside a docker container. All later steps run in the interactive shell.
docker run -it --rm <imageName> /bin/bash

### Clone source-build repo
git clone https://github.com/dotnet/source-build
cd source-build

### Build source-build to create a tarball
./build-source-tarball.sh ../tarball

### Build the tarball
cd ../tarball
./build.sh

### Clean up the container when you're done working with the build results
exit
```

Reply:
I added this as an "interactive docker container" section and left the other as an example with a docker volume.